synapse/util/manhole.py

```diff
@@ -98,7 +98,9 @@ def manhole(settings: ManholeConfig, globals: Dict[str, Any]) -> ServerFactory:
         SynapseManhole, dict(globals, __name__="__console__")
     )
 
-    factory = manhole_ssh.ConchFactory(portal.Portal(rlm, [checker]))
+    # type-ignore: This is an error in Twisted's annotations. See
+    # https://github.com/twisted/twisted/issues/11812 and /11813 .
+    factory = manhole_ssh.ConchFactory(portal.Portal(rlm, [checker]))  # type: ignore[arg-type]
```

Comment:
Urgh, the error is only introduced in twisted trunk, so adding this triggers an error on the current build.

Maybe I can use a `cast` instead.

Reply:
I opted just to tell mypy to relax the unused-ignore check for this file. (Simpler, and I'm pretty sure it can complain about a redundant cast sometimes.)

Reply:
Hmm, I'm not sure it has an error code:

```
synapse/util/manhole.py:103: error: Unused "type: ignore" comment
```

Maybe I can just use a naked # type[ignore]?

Reply:
Worth a try, if not :shipit:

Reply:
No luck, I'm afraid. :ship: ping it